variants/fcos/okd-rpms/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM release:artifacts as artifacts
```

Comment:
we want to avoid adding new Dockerfiles here.

Reply:
Right, we don't need those. `okd-rpms` is useful for Assisted Installer but not usable for SCOS installs

Reply:
just for my understading: will release:artifacts and release:installer artifacts be converged further?